itest/litd_mode_integrated_test.go

```diff
@@ -513,6 +539,58 @@ func runGRPCWebAuthTest(t *testing.T, hostPort, uiPassword, grpcWebURI string) {
 	require.Contains(t, body, "grpc-status: 0")
 }
 
+// runRESTAuthTest tests authentication of the given REST interface.
+func runRESTAuthTest(t *testing.T, hostPort, uiPassword, macaroonPath, restURI,
```

Comment:
possibly a silly question: but why are these REEST tests dependent on the "support supermac in remote lnd mode" changes?

Reply:
Not silly at all! I noticed we didn't test REST yet but didn't want to bloat the existing itest PR (since it already received some review). So I added that commit here. But there is no dependency whatsoever to the other changes. I can move the commit if you prefer.

Reply:
Too late to move to the itest PR now, I guess :sweat_smile: